src/nbl/video/CVulkanLogicalDevice.cpp

```diff
@@ -9,6 +9,37 @@
 namespace nbl::video
 {
 
+core::smart_refctd_ptr<IGPUSemaphore> CVulkanLogicalDevice::createSemaphore(IGPUSemaphore::SCreationParams&& params)
+{
+    VkImportSemaphoreWin32HandleInfoKHR importInfo = { VK_STRUCTURE_TYPE_IMPORT_SEMAPHORE_WIN32_HANDLE_INFO_KHR };
```

Comment:
this is currently an unused variable correct?

Reply:
Correct